src/components/new-request/search.vue

```diff
@@ -9,48 +9,62 @@
     <v-row class="mt-6">
       <!-- Request Action -->
       <v-col cols="12" md="6" class="py-0">
-        <v-select
-          filled
-          label="Select an Action"
-          :error-messages="getErrors.includes('request_action_cd') ? 'Please select an action' : ''"
-          :items="requestActions"
-          item-value="[group,value]"
-          :menu-props="{ bottom: true, offsetY: true, maxHeight: 423 }"
-          @change="setClearErrors(null); onRequestActionChange($event)"
-          return-object
+        <v-tooltip
+          top
+          content-class="top-tooltip"
+          transition="fade-transition"
+          :disabled="!showRequestActionTooltip || isMobile"
         >
-          <!-- FUTURE: use "selection" slot to format the selected business -->
-          <!-- <template #selection="{ item }">
-            <div class="font-weight-bold text-truncate">{{ item.text }}</div>
-            <div class="text-subtitle-1">{{ item.subtext }}</div>
-          </template> -->
-
-          <template #item="{ item }">
-            <v-list-item-content
-              v-if="item.isHeader"
-              class="group-header px-4 py-5"
-              @click.stop="toggleActionGroup(item.group)"
-            >
-              <div class="d-flex justify-space-between align-center">
-                <p class="mb-0 mr-4" :class="{'app-blue': item.group === activeActionGroup}">{{ item.text }}</p>
-                <v-icon color="primary">
-                  {{ item.group === activeActionGroup ? 'mdi-chevron-up' : 'mdi-chevron-down' }}
-                </v-icon>
-              </div>
-            </v-list-item-content>
-
-            <!-- render but conditionally hide disabled list items, so that the v-select
-            continues to display the current selection even when a different group is active -->
-            <v-list-item-content
-              v-else
-              class="group-item pl-8 pr-4 py-4"
-              :class="{ 'hide-me': item.disabled }"
-            >
-              <div class="font-weight-bold">{{ item.text }}</div>
-              <div>{{ item.subtext }}</div>
-            </v-list-item-content>
+          <template v-slot:activator="scope">
+            <div v-on="scope.on">
+              <v-select
+                id="search-type-options-select"
+                class="request-action-select"
```

Comment:
I restored the id (from the original code) even though it's applied to the HTML input element, which is pretty useless for us right now.

The class is used as a selector to find the inner selection. I tried using a ref here instead -- which is equivalent to selecting this item -- but then crawling its children to find the inner selection was messy/fragile, whereas `querySelector()` is simple and probably more robust.